web-common/src/components/data-graphic/marks/MultiMetricMouseoverLabel.svelte

```diff
@@ -134,17 +134,19 @@ It is probably not the most up to date code; but it works very well in practice.
   $: if (container && locations && $xScale && $yScale) {
     clearTimeout(transitionalTimeoutForCalculatingLabelWidth);
     transitionalTimeoutForCalculatingLabelWidth = setTimeout(() => {
-      labelWidth = Math.max(
-        ...Array.from(container.querySelectorAll(".widths")).map(
-          (q: SVGElement) => q.getBoundingClientRect().width
-        )
-      );
+      if (container) {
```

Comment:
Quickly went through this. We are already checking if `container` is valid on line 134 right? Adding another check in the same block looks counter-intutive.